bot/pkg/bsky/bsky.go

```diff
@@ -31,10 +32,13 @@ type SearchResponse struct {
 	Feed []FeedItem `json:"feed"`
 }
 
-func FetchPosts(client *gsheets.Client) {
+func FetchPosts(contentType post.NYTContentType, client *gsheets.Client) {
 
 	resp, err := http.Get(bskyFeedUrl)
 
+	// temporary workaround to pass correct type into google sheet write (string, rather than content type).
+	//TODO: remove/refactor this once sheets write logic is finalized
+
```

Comment:
What is the temporary workaround exactly? This appears to refer to lines that already existed 😄

Reply:
I picked this PR up and had to do a slight refactor in order to merge main. I think I accomplished the TODO. Will remove! Good catch

Reply:
This comment was from one of my previous handoff commits from the AM session – there was a typecast to string for an earlier iteration of our gsheets client. Probably good to remove to avoid confusion!